src/main/java/com/tabnine/binary/fetch/BootstrapperSupport.java

```diff
@@ -31,13 +31,17 @@ private static BinaryVersion savePreferredBootstrapVersion(BinaryVersion version
 
     private static Optional<BinaryVersion> locateLocalBootstrapSupportedVersion(LocalBinaryVersions localBinaryVersions) {
         return minimalBootstrappedVersion().flatMap(
-                version ->
-                        localBinaryVersions
+                version -> {
+                    Optional<BinaryVersion> activeVersion = localBinaryVersions.activeVersion();
+                    if (activeVersion.isPresent()) return activeVersion;
+
+                    return localBinaryVersions
```

Comment:
use `Optional.orElseGet`

Reply:
cannot because it still might be `Optional.empty()`